static/js/workshop.js

```diff
@@ -0,0 +1,20 @@
+// My dislike of jQuery is overshadowed by the ease of this use-case
+$(document).ready(function() {
+    var publicWorkshop = $('#id_public');
+    var privateCodeRow = $('.field-private_code');
+    var privateCodeInput = $('#id_private_code');
+    var privateCodeUrlSpan = $('#pcode');
```

Comment:
It would be great to have the same helper URL show up in the Inline Rate Editor:

![image](https://cloud.githubusercontent.com/assets/274668/18528063/370dffaa-7a7c-11e6-91c6-b0c4f63df8b1.png)